src/shared/ui/subtitle/subtitle.tsx

```diff
@@ -1,10 +1,45 @@
-import './subtitle.scss';
+import { HTMLAttributes } from 'react';
+import { type VariantProps, cva } from 'class-variance-authority';
+import classNames from 'classnames/bind';
 
-interface SubtitleProps {
-  text: string;
-  type?: string;
-}
+import styles from './subtitle.module.scss';
 
-export const Subtitle = ({ text, type = '' }: SubtitleProps) => (
-  <div className={`subtitle ${type}`}>{text}</div>
-);
+type SubtitleProps = Pick<HTMLAttributes<HTMLHeadingElement>, 'className' | 'children'> &
+  VariantProps<typeof subtitleVariants>;
```

Comment:
I have seen the the same problem in our codebase before, but are we sure to use type ```SubtitleProps```, which includes type of variable ```subtitleVariants```, before definition of this variable?